apps/dg/controllers/document_controller.js

```diff
@@ -1295,26 +1279,27 @@ DG.DocumentController = SC.Object.extend(
               dataContext: iContext
             }),
             tTitle = iTitle || (iContext ? iContext.get('name') : 'DG.DocumentController.caseCardTitle'.loc()),
-            tModel, tRestoredLayout = iLayout;
-        if (!iComponent) {
-          tModel = this.configureComponent(iComponent, {
+            tComponent = iComponent, tRestoredLayout = iLayout;
```

Comment:
I did a little bit of refactoring here just to make the code clearer to me. There was inconsistent use of `iComponent'/`tComponent` and the variable named `tModel` actually held a Component[Model] but the name led me to think it was a `ContentModel`.